### PR TITLE
Validations

### PR DESCRIPTION
**Pull request by BiancaMe**
Add the following validations:

For the User model:
-Name must not be blank.
-PostsCounter must be an integer greater than or equal to zero.

For the Post model:
-Title must not be blank.
-Title must not exceed 250 characters.
-CommentsCounter must be an integer greater than or equal to zero.
-LikesCounter must be an integer greater than or equal to zero.

Add unit specs for all of your models' methods and validations.

### CODE REVIEW

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'Bianca', photo: 'photo.com.uy', bio: 'developer from Uruguay') }
+
+  before { subject.save }
+
+  it 'Name must not be blank.' do
+    expect(subject).to be_valid
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
```
Comment (OyePriscilla) [inline spec/models/user_spec.rb]: - [ ] In accordance to the project requirement, kindly implement test for the user's model's method highlighted from your codebase. 👇 

```
 def recents(limit = 3)
    posts.order(created_at: :desc).limit(limit)
  end
```

Untitled

File: spec/models/post_spec.rb [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+RSpec.describe Post, type: :model do
+  user = User.new(name: 'Bianca', photo: 'photo.com.uy', bio: 'developer from Uruguay')
+  user.save
+  post = Post.new(author: user, title: 'First Post', text: 'Welcome to my first post')
+  comment = Comment.new(text: 'Beatifull Post', post:, user:)
+  comment.save
+  like = Like.new(post:, user:)
+  like.save
+
```
Comment (OyePriscilla) [inline spec/models/post_spec.rb:11]: - [ ] In accordance to the project requirement, kindly implement test for the post's model's method highlighted from your codebase. 👇 

```
 def recents(limit = 5)
    comments.order(created_at: :desc).limit(limit)
  end
```

Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @BiancaMe 👏 👏 👏 👏, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Implemented validations for models. ✔️ 
Implemented unit tests for models. ✔️ 
All tests passed. ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — COMMENTED** [review]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @BiancaMe,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly submit a fresh PR, without reviewer comments, since this was not approved for the previous week.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request closed without merging**